package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "dist/generateBaseFieldsInserts.js",
   "scripts": {
-    "lint": "eslint ./src --ext .ts",
+    "lint": "eslint ./src --ext .ts --max-warnings=0",
```

Comment:
Rather than using flags here what do you think about creating an .eslintrc file for eslint configuration and moving ALL of these settings to it?

I find that to be a bit more legible (and also more potentially modular / easy to swap out for various environments)

Reply:
I looked, and it turns out this particular option is command-line only! The PR that introduced it discussed the complexity of supporting such a configuration option in the context of sub-folder ESLint config files.

Reply:
Ahh yeah that makes sense -- and I also see the logic (the context of whether a warning should be a blocker is based on the context of the run itself, not the project).

Thanks!